local.c

```diff
@@ -109,6 +109,12 @@ static const char * const device_attrs_blacklist[] = {
 	"uevent",
 };
 
+static const char * const buffer_attrs_reserved[] = {
+	"length",
+	"watermark",
```

Comment:
Would we want to also expose watermark? I don't see an existing mechanism to interact with this attribute in libiio